src/index.js

```diff
@@ -17,17 +17,17 @@ gisida.version = require('../package.json').version;
 gisida.initStore = require('./store/initStore').default;
 gisida.loadLayers = require('./store/initStore').loadLayers;
 gisida.reducerRegistry = require('./store/reducerRegistry').default;
-gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').default;
+gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').createMapReducer;
```

Comment:
Why / when did this change?

Reply:
@cKellyDesign The reducers.js file was split into various modules which currently exist on separate files. here is the current createmapreducer https://github.com/onaio/gisida/blob/staging/src/store/reducers/createMapReducer/createMapReducer.js